src/Hooks/GutenbergCleanup.php

```diff
@@ -44,7 +43,7 @@ public function removeGutenbergFootprint( $check, $postId, $metaKey, $metaValue
 
 			// $update :: int -> string -> bool
 			$update = curryN( 2, function( $postId, $meta ) {
-				Post::updateMeta( $postId, WPML_Elementor_Data_Settings::META_KEY_DATA, $meta );
+				update_metadata( 'post', $postId, WPML_Elementor_Data_Settings::META_KEY_DATA, $meta );
```

Comment:
@grey-rsi Please add a comment here or in the test about why we need to use `update_metadata( 'post' ... )` instead of `Post::updateMeta` or `update_post_meta`.

Reply:
@brucepearson done

Reply:
Are you running phpcs there?

Comments should end with a full stop and there has to be space between `//` and the comment.

I am always intrigued at why these formatting issues aren't caught automatically for you.

Reply:
Fixed